common/locales/en.json

```diff
@@ -295,6 +295,11 @@
         "subtitleAlignment": "Subtitle Alignment",
         "subtitleAlignmentBottom": "Bottom",
         "subtitleAlignmentTop": "Top",
+        "subtitleBlur": "Subtitle blur",
+        "subtitleBlurTrack1": "Track 1",
+        "subtitleBlurTrack2": "Track 2",
+        "subtitleBlurTrack3": "Track 3",
+        "subtitleBlurDescription": "Hides selected subtitle tracks by blurring them, can be unblurred on mouse hover",
```

Comment:
Nitpick: `Hides selected subtitle tracks by blurring them. Can be un-blurred on mouse hover.`